src/Calculator/RefactoredTokenizer.js

```diff
@@ -0,0 +1,67 @@
+function Tokenizer() {
+  function ValueToken(value, type) {
+    this.type = type;
+    this.value = Number(value);
+  }
+
+  function OperationToken(value1, value2, type) {
+    this.type = type;
+    this.value1 = value1;
+    this.value2 = value2;
+  }
+
+  const tokenize = input => {
+    const array = input.match(/(\d+)?(\.?\d+)|\+|\-|\*|\//g);
+    const symbol = {
+      "+": "sum",
+      "-": "rest",
+      "*": "mult",
+      "/": "div"
+    };
+    let accumulator = "";
+
+    for (let i = 0; i < array.length; i++) {
+      if (
+        array[i] !== "+" &&
```

Comment:
You could use Array.includes
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes